scripts/release.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+set -e
+
+REPOSITORY="${REPOSITORY:-git@github.com:etcd-io/etcd-operator.git}"
+REGISTRY="${REGISTRY:-gcr.io/etcd-development/etcd-operator}"
+CONTAINER_TOOL="${CONTAINER_TOOL:-docker}"
+
+function main {
+  if [ -z "$1" ]; then
+    echo "Usage: $0 <release version>"
+    exit
+  fi
+
+  local version="v${1#v}"
+  local image="${REGISTRY}:${version}"
+
+  build_and_commit_dist_files "${image}"
+
+  if ! tag_repository "$version"; then
+    echo "Failed to tag repository"
+    exit 1
+  fi
+
+  build_and_push_image "${image}"
```

Comment:
Minor comment: I think we should build & push images firstly, and generate the `dist/install.yaml` later because it refers to (or depends on) the image.

Reply:
That sounds good. I moved the order many times and was never convinced. Your point makes sense. I'll update it.

Reply:
Done.